packages/babel-plugin-minify-dead-code-elimination/src/index.js

```diff
@@ -710,6 +715,10 @@ module.exports = ({ types: t, traverse }) => {
           traverse.clearCache();
           path.scope.crawl();
 
+          if (!isEvalScopesMarked(path.scope)) {
+            markEvalScopes(path);
```

Comment:
I didn't put isMarked inside markEvalScopes because if something is changed after marking - we need to remark.